src/scripting/flash/system/flashsystem.cpp

```diff
@@ -176,6 +197,117 @@ ASFUNCTIONBODY_ATOM(Capabilities,_getServerString)
 	*/
 	ret = asAtomHandler::fromString(sys,res);
 }
+
+ASFUNCTIONBODY_ATOM(Capabilities,_hasAudio)
+{
+    LOG(LOG_NOT_IMPLEMENTED,"Capabilities.hasAudio always returns true");
+    ret = asAtomHandler::fromBool(true);
+}
```

Comment:
We should return true only if we are compiling with ffmpeg enabled (#ifdef ENABLE_LIBAVCODEC)